name-service/src/main/resources/project-stages.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Same as above, separate to individual `project-*.yml` files. Or in this case, just remove the `project: \n stage: local` stuff and rename the file to `project-local.yml`.

Reply:
done